cluster/log-dump/logexporter-daemonset.yaml

```diff
@@ -36,7 +36,7 @@ spec:
     spec:
       containers:
       - name: logexporter-test
-        image: gcr.io/k8s-testimages/logexporter:v0.1.2
+        image: k8s.gcr.io/logexporter:v0.1.1
```

Comment:
I suggest that instead of reverting to older image, let's over-write the current one with a debugged version (to avoid keeping the buggy version exposed). This is against best practice, but I guess logexporter is more of a test thing that pretty much only we use.

Reply:
OK